app/src/main/java/cm/aptoide/pt/socialMedia/SocialMediaNavigator.kt

```diff
@@ -0,0 +1,32 @@
+package cm.aptoide.pt.socialMedia
+
+import android.content.Context
+import cm.aptoide.aptoideviews.socialmedia.SocialMediaView
+import cm.aptoide.pt.R
+import cm.aptoide.pt.link.CustomTabsHelper
+import cm.aptoide.pt.themes.ThemeManager
+
+class SocialMediaNavigator(val context: Context, val themeManager: ThemeManager) {
+
+  companion object {
+    var FACEBOOK_URL = "https://pt-pt.facebook.com/aptoide/"
+    var TWITTER_URL = "https://twitter.com/aptoide"
+    var INSTAGRAM_URL = "https://www.instagram.com/aptoideteam/"
+  }
+
+  fun navigateToSocialMediaWebsite(socialMediaType: SocialMediaView.SocialMediaType) {
+    val socialMediaUrl: String = mapSocialMediaTypeToUrl(socialMediaType)
+    CustomTabsHelper.getInstance()
+        .openInChromeCustomTab(socialMediaUrl, context,
+            themeManager.getAttributeForTheme(
+                R.attr.colorPrimary).resourceId)
```

Comment:
In my opinion we should have an ExternalNavigator which would unduplicate this logic. We seem to be doing this type of navigation accross the app but there is no standardization.